backend/coins/btc/db/transactionsdb/transactionsdb_test.go

```diff
@@ -83,6 +87,93 @@ func getRawValue(tx *Tx, bucketKey string, key []byte) []byte {
 	return tx.tx.Bucket([]byte(bucketKey)).Get(key)
 }
 
+// TestTxSerialization checks that Tx marshaling/unmarshaling work with both current and legacy
+// chainhash.Hash marshaling methods.
+// see https://github.com/btcsuite/btcd/pull/2025.
+func TestTxSerialization(t *testing.T) {
+	db := getDB()
+	defer func() {
+		if err := db.Close(); err != nil {
+			panic(err)
+		}
+	}()
+	dbTx, err := db.Begin(true)
+	if err != nil {
+		panic(err)
+	}
+	defer dbTx.Rollback()
+
+	// create a new Tx
+	_, addressChain := addressesTest.NewAddressChain(
+		func(address *addresses.AccountAddress) (bool, error) {
+			return false, nil
+		},
+	)
+	addresses, err := addressChain.EnsureAddresses()
+	require.NoError(t, err)
+	address := addresses[0]
```

Comment:
you could skip all this and simply use a dummy pkScript

Reply:
Good point! Done, thanks